Models/user.js

```diff
@@ -6,10 +6,10 @@ const model = mongoose.model("users", new Schema({
   username: String,
   password: String,
   email: String,
-  phone: String
+  phone: String,
+  responders: [{_id:false, id: String}]
```

Comment:
Why two ids? Also an array of string (string being the id) would work just as well?

Reply:
the false is to prevent an additional ID from being inserted (without it there would be an objectID for individual responder ID inserted for somewhat reason)

Reply:
_id is the automatic generated object ID when inserting into mongoDB, id is the manual one I chose to create

Reply:
so an example of a responder value for an arbitrary user would be:
responders: [{ id: "some string" }, { id: "some other string" }]
Please confirm

Reply:
yup